### ISSUE TITLE
Water Sensors: FQU Indigo

### ISSUE THREAD

**Issue by kldriver** [precision-sustainable-ag/On-farm-Protocols#305]
**What's wrong**
R1C never made an initial connection. R2C is missing a deep sensor. All other sensors at this site are online and transmitting regularly. 

**What done looks like**
Indigo tech go to the site and check power and interconnectivity.

**Comment by elhenriq** [comment]
as you mentioned, one node is not communicating. first thing I would do is check on status of the node, make sure it was power on, check the battery was installed properly and that is fully charged. after that, I would reset the entire farm to make they all communicate again. additionally, your Rep 2 Cover the deep sensor is not communicating. please check that the cables have not been cut when installed. if in fact they were cut, please call the hotline. if they have not been cut, check the junction box and make sure all cables look are good there.

**Comment by elhenriq** [comment]
Tyler called today, he went to visit this farm to get the node reconnected to the gateway. after trying to get the nodes working his gateway lost connection to the network and would not connect back to them again, we tried testing cellular connection but was not successful. we left all four nodes talking to the gateway and will wait to see if the gateway will connect to the network sometime. he moved the gateway to a better location to get it to work.

also, we change the address on rep 2 cover to get the sensor string working again. we also tried to download the data off the cover rep 1 node, but there was no data to download.
